src/main/webapp/i18n/de/quizExercise.json

```diff
@@ -61,7 +61,8 @@
             "isOpenForPractice": "Zum Üben freigegeben",
             "isPlannedToStart": "Geplante Startzeit",
             "openForPractice": "Zum Üben Freigeben",
-            "startNow": "Jetzt Starten",
+            "startQuiz": "Quiz Starten",
```

Comment:
You could write `Quiz starten` or `Starte das Quiz` (but I think the first version is preferable because it has the same length as the english translation)